drivers/ds18/ds18.c

```diff
@@ -25,7 +22,8 @@
 
 #include "log.h"
 #include "periph/gpio.h"
-#include "xtimer.h"
+#include "ztimer.h"
+#include "time_units.h"
```

Comment:
Maybe I missed it, but I didn't see anything new using this header.

Reply:
I moved include to `drivers/ds18/ds18_internal.h`.